api4/api.go

```diff
@@ -139,6 +139,8 @@ type Routes struct {
 	InsightsForUser *mux.Router // 'api/v4/users/me/top'
 
 	Usage *mux.Router // 'api/v4/usage'
+
+	Events *mux.Router // 'api/v4/events'
```

Comment:
I rather have this under the `system` path, would it make sense to you?

Reply:
yes, sounds reasonable